lib/PEX.ts

```diff
@@ -106,7 +106,7 @@ export class PEX {
     let presentationSubmission = opts?.presentationSubmission;
     let presentationSubmissionLocation =
       opts?.presentationSubmissionLocation ??
-      (Array.isArray(presentations) || !CredentialMapper.isW3cPresentation(wrappedPresentations[0].presentation)
+      ((Array.isArray(presentations) && presentations.length > 1) || !CredentialMapper.isW3cPresentation(wrappedPresentations[0].presentation)
```

Comment:
This change is incorrect. If we have an array it should be external.

Reply:
Yeah but the problem is that the presentation(s) are created in PEX along the way. So now it simply cannot deduce it anymore for the case when there is one presentation. Given we should be sending in the location arg for OID4VP anyway, we can leave it for now. But next time you encounter a situation like this please discuss first.